sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample7_AnalyzeHealthcareEntitiesBatchConvenienceAsync.cs

```diff
@@ -34,50 +34,69 @@ public async Task HealthcareBatchAsync()
 
             string document2 = "Prescribed 100mg ibuprofen, taken twice daily.";
 
-            List<string> batchInput = new List<string>()
```

Comment:
so convenience means we are using the list of strings instead of list of `TextDocumentInput`. looks like if u just swap the name for with `Sample7_AnalyzeHealthcareEntitiesBatchAsync` it should work